### PR TITLE
Using one policy for tests

### PR DESCRIPTION
**Pull request by orenbm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,21 +22,35 @@ If you are using Maven to manage your project's dependencies, you can run `mvn i
...
+
 ```bash
 CONJUR_ACCOUNT=accountName
 CONJUR_CREDENTIALS=username:apiKey
 CONJUR_APPLIANCE_URL=http://conjur
 ```
 
+In addition, you will need to load a Conjur policy. Save this file as `root.yml`:
+
+```java
```
Comment (jvanderhoof) [inline README.md]: should this be `yaml` instead of `java`?

File: README.md [hunk 2]
```diff
@@ -54,7 +68,8 @@ where the Conjur object is logged in to the account & ready for use.
 
 ### Variable Operations
 
-A Variable is an access-controlled list of encrypted data values. The values in a Variable are colloquially known as ?secrets?.
+A Variable is an access-controlled list of encrypted data values. The values in a Variable are colloquially known as
```
Comment (jvanderhoof) [inline README.md]: This isn't quiet right.  A variable is a named reference.  The most common thing a variable references is a secret. A variable can have one or more (up to 20 secrets) associated with it, and ordered in reverse chronological order.

File: test-policy/test.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+- &variables
+                                - !variable testVariable
```
Comment (jvanderhoof) [inline test-policy/test.yml]: Looks like spacing may have gotten a bit off here.

File: README.md (hunk 1 above)
Comment (orenbm) [inline README.md]: You're right

File: README.md (hunk 2 above)
Comment (orenbm) [inline README.md]: Weird, I took this from some documentation but I can't find it.

**Pull request merged**